.github/workflows/richnav.yml

```diff
@@ -0,0 +1,30 @@
+name: Rich Navigation Indexing
+
+on: [push, pull_request]
+
+env:
+  postgis_version: 3
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+
+jobs:
+  build:
+    runs-on: windows-2019
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: ${{ env.dotnet_sdk_version }}
```

Comment:
Looks like this is based on our build.yml, but the env var doesn't exist here... Do we actually need .NET Core SDK and dotnet build below?